test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.util;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+/**
+ * Class to generate typical test persons
+ */
+public class TypicalPersons {
+
+    public static Person amy, bill, candy, donald, grace;
```

Comment:
@damithc Addressbook-Level4 has the same problem too. I'll open up an issue there.